modules/hydra-master-main.nix

```diff
@@ -79,30 +94,19 @@ in {
       <github_authorization>
         input-output-hk = ${builtins.readFile ../static/github_token}
       </github_authorization>
-      <githubstatus>
-        jobs = serokell:iohk-ops.*:required
-        inputs = jobsets
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
-      <githubstatus>
-        jobs = Cardano:cardano-ledger.*:required
-        inputs = chain
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
-      <githubstatus>
-        jobs = Cardano:cardano-ledger-specs.*:required
-        inputs = cardano-ledger-specs
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
-      <githubstatus>
-        jobs = serokell:cardano.*:required
-        inputs = cardano
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
+
+      ${mkStatusBlocks [
+        { jobset = "iohk-ops"; inputs = "jobsets"; }
+        { jobset = "cardano-ledger"; inputs = "cardano-ledger"; }
+        { jobset = "cardano-ledger-specs"; inputs = "cardano-ledger-specs"; }
+        { jobset = "cardano"; inputs = "cardano"; }
```

Comment:
this jobsset, `cardano.*` will also match jobs from `cardano-ledger.*` (which in turn matches jobs from `cardano-ledger-specs.*` ). Would that create multiple notifications?

Reply:
the notify plugin in hydra will stop at the first regex to match, and ignore all others that follow

Reply:
ok thanks! so we need `cardano-ledger-specs` before `cardano-ledger`, then?